src/app/build.gradle

```diff
@@ -0,0 +1,85 @@
+apply plugin: 'com.android.application'
+apply plugin: 'android-apt'
+apply from: '../jacoco.gradle'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 23
+        applicationId "br.com.italoromeiro.arctouchtest"
+        testApplicationId 'br.com.italoromeiro.arctouchtest.test'
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            debuggable true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+
+        debug {
+            applicationIdSuffix '.dev'
+            testCoverageEnabled true
+            ext.betaDistributionGroupAliases = "android-testers"
+        }
+    }
+
+    testOptions {
+        resultsDir = "${project.buildDir}/foo/results"
+    }
+}
+
+apt {
+    arguments {
+        resourcePackageName android.defaultConfig.applicationId
+        androidManifestFile variant.outputs[0]?.processResources?.manifestFile
+    }
+}
+
+configurations.all {
+    resolutionStrategy.force 'com.android.support:support-annotations:23.4.0'
+    resolutionStrategy.force 'com.android.support:appcompat-v7:23.4.0'
+    resolutionStrategy.force 'com.android.support:design:23.4.0'
+    resolutionStrategy.force 'com.android.support:recyclerview-v7:23.4.0'
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+
+    // UNIT TESTS
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'org.mockito:mockito-core:2.0.100-beta'
+
+    // ANDROID TESTS
+    androidTestCompile 'com.android.support.test:runner:0.5'
+    androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
+    androidTestCompile 'com.android.support.test.espresso:espresso-contrib:2.2.2'
+    androidTestCompile 'com.squareup.okhttp3:mockwebserver:3.4.1'
+    androidTestCompile 'com.squareup.retrofit2:retrofit-mock:2.1.0'
+
+    // GOOGLE APIs
+    compile 'com.android.support:appcompat-v7:23.4.0'
+    compile 'com.android.support:design:23.4.0'
+    compile 'com.android.support:recyclerview-v7:23.4.0'
+    compile 'com.google.android.gms:play-services-location:9.4.0'
+    compile 'com.google.android.gms:play-services-maps:9.4.0'
+
+    // ANDROID ANNOTATIONS
```

Comment:
What do you think are the main advantages of using Android Annotations? And what do you think are the downsides?